README.md

```diff
@@ -91,6 +91,7 @@ instance. There are several ways to provide this information to `lab`:
 3. local configuration file in [Tom's Obvious, Minimal Language (TOML)](https://github.com/toml-lang/toml): `./lab.toml`;
     - No other config files will be used as overrides if a local configuration file is specified
 4. user-specific configuration file in TOML: `~/.config/lab/lab.toml`.
+    - an example of user-specfic configurations in lab.toml can be found here.  As can be seen in the example file, any command option can be set in the configuration file.
```

Comment:
Isn't a link to `docs/lab.toml` missing in the word `here`?

Reply:
Oops.  Sorry, this was meant to be a draft!  I was going to play with the output a bit ... but yes, it will be a link.

Reply:
Oh, my bad. I overlooked the "draft" state!
Sorry for that :))